tools/mksurfdata_map/src/mksurfdat.F90

```diff
@@ -1234,13 +1222,24 @@ program mksurfdat
 	  call check_ret(nf_sync(ncid), subname)
 
        end do   ! end of read loop
-
+       do n = 1, ns_o
+          do k =1, numurbl
+             if (pcturb_max(n,k) > 1.e-3_r8) then 
```

Comment:
- [ ] I think it could be problematic to have a hard-coded threshold of 1e-3 here: I feel like this should just check for `pcturb_max > 0`. (There would be a problem if hasurban is set to 0 but pcturb is large enough to trigger the attempted creation of an urban landunit. Maybe the solution is to do a truncation to 0 of too-small urban (e.g., less than 1e-3) at some point prior to this, and then this check here can just check for > 0?) However, this may be irrelevant if you go with my suggestion elsewhere of getting rid of HASURBAN and just writing PCT_URBAN_MAX to the landuse timeseries file – in which case this whole block of code would be removed.

Reply:
Yes, this block of code is deleted since we now use PCT_URBAN_MAX.